hydra/test_utils/config_source_common_tests.py

```diff
@@ -10,6 +11,15 @@
 
 
 class ConfigSourceTestSuite:
+    def skip_overlap_config_path_name(self) -> bool:
+        """
+        Some config source plugins do not support config name and path overlap, for example:
```

Comment:
Extended the commend to explain what this function is and how it should be used.